resources/views/projects/details.blade.php

```diff
@@ -158,6 +219,41 @@ function showTaskModal() {
 		$("#new-task-modal").modal("show");
 	}
 
+	function showMilestoneModal(action) {
+		switch (action) {
+			case 'create':
```

Comment:
You could reduce all of this method to:

```suggestion
			$(`#${action}-milestone-modal`).modal('show');
```